Gir_Gtk.toml

```diff
@@ -357,3 +357,10 @@ status = "generate"
 [[object]]
 name = "Gtk.PaperSize"
 status = "generate"
+
+[[object]]
+name = "Gtk.WidgetPath"
+status = "generate"
+ignored_functions = [
+    '''iter_has_q\w+''',
```

Comment:
Does this need triple quotes? It seems they're necessary if there's a single quote inside the string.